lib/schema-evolution-manager/apply_util.rb

```diff
@@ -10,6 +10,7 @@ def initialize(db, opts={})
 
       @db = Preconditions.assert_class(db, Db)
       @scripts = Scripts.new(@db, Scripts::SCRIPTS)
+      @pgpass_file = Preconditions.assert_class(pgpass_file, Tempfile)
```

Comment:
Currently I am writing to it, so unless we change it to the EV I need to pass the file